### PR TITLE
mamali/update-affiliates-and-add-code-of-conduct

### PR DESCRIPTION
**Pull request by reza-fs**

### CODE REVIEW

File: src/templates/static/legal/code_of_conduct.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+const CodeOfConduct = () => (
+    <div>
+        <h2 data-anchor='code-of-conduct'>{it.L('Code of Conduct for Business Partners')}</h2>
+        <h2 data-anchor='introduction'>{it.L('Introduction')}</h2>
+        <p>{it.L(
+            'Binary.com is one of the oldest and most successful online trading companies in the world. Our commitment to high standards of ethics is one of the key factors that will ensure our continuing success.  Our Business Principles summarise the values that underpin our work.')}
```
Comment (negarn) [inline src/templates/static/legal/code_of_conduct.jsx]: instead of `Binary.com` please use `[_1]` with `it.website_name` throughout

File: src/templates/static/legal/tac_sections/affiliate.jsx [hunk 2]
```diff
@@ -1,158 +1,358 @@
-import React from 'react';
-import ListNested from '../../../_common/components/list_nested.jsx';
+import React       from 'react';
+import ListNested  from '../../../_common/components/list_nested.jsx';
 
 const Affiliate = () => (
     <div>
-        <h2 data-anchor='affiliate-terms-and-conditions'>{it.L('[_1]\'s Affiliate Programme terms and conditions', it.website_name)}</h2>
+        <h2 data-anchor='affiliate-terms-and-conditions'>{it.L(
+            '[_1]\'s Affiliate Programme terms and conditions',
```
Comment (negarn) [inline src/templates/static/legal/tac_sections/affiliate.jsx]: do we still need to repeat `terms and conditions` here now that it's moved to be part of t&c?

File: src/templates/static/legal/tac_sections/affiliate.jsx [hunk 3]
```diff
@@ -1,158 +1,358 @@
...
-                    { text: it.L('The Affiliate represents, warrants, and undertakes that the Affiliate\'s website or promotion shall contain no material that is defamatory, pornographic, unlawful, harmful, threatening, obscene, harassing, or racially, ethnically, or otherwise objectionable or discriminatory, violent, politically sensitive, or otherwise controversial or in breach of any third party rights and shall not link to any such material.') },
+                    {
+                        text: it.L(
+                            'Each party to this Agreement represents and warrants to the other that it has, and will retain throughout the term of this Agreement, all right, title, and authority to enter into this Agreement, to grant to the other party the rights and licences granted in this Agreement, and to perform all of its obligations under this Agreement.'),
```
Comment (negarn) [inline src/templates/static/legal/tac_sections/affiliate.jsx]: honestly this isn't any more readable than how it used to be, if you look at the PR diff. can we revert these changes so it matches the rest of the files instead?

Comment (reza-fs) [reply src/templates/static/legal/tac_sections/affiliate.jsx]: Yes.

File: src/templates/static/legal/tac.jsx [hunk 4]
```diff
@@ -27,6 +29,11 @@ const Tac = () => (
                             { id: 'legal-mt',     text: it.L('MetaTrader') },
                         ],
                     },
+                    {
+                        id  : 'affiliate',
+                        text: it.L('Affiliate Programme', it.website_name),
```
Comment (negarn) [inline src/templates/static/legal/tac.jsx]: what is it.website_name supposed to replace?

Comment (reza-fs) [reply src/templates/static/legal/tac.jsx]: Oh geez. sorry for that.
